kmk/ble.py

```diff
@@ -14,19 +14,20 @@
 
 
 class BLEHID(AbstractHID):
-    def post_init(self):
+    def post_init(self, ble_name='KMK Keyboard', **kwargs):
         self.devices = {}
 
         hid = HIDService()
 
         advertisement = ProvideServicesAdvertisement(hid)
         advertisement.appearance = BLE_APPEARANCE_HID_KEYBOARD
-        scan_response = Advertisement()
-        scan_response.complete_name = 'KMK Keyboard'
 
         ble = BLERadio()
+        ble.name = ble_name
+        # ble.tx_power = 2
```

Comment:
generally commented-out code doesn't stick around in the tree, but I'm not going to block merge over it. more an FYI for the future.

Reply:
Yea, it's a placeholder. TX power is not yet supported, but it should be in the near future